### DISCUSSION TITLE
Break overlay shows F grade while exiting gameplay

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#22051] [Q&A] [answered]
As title. Probably not intended.

It seems to apply to all cases (i.e. gameplay/replay/autoplay). Doesn't appear to be map-specific.

https://user-images.githubusercontent.com/43073074/211062619-0421697c-90e4-46e7-82e1-9a18dc2eb4eb.mp4

**Comment by peppy** [comment, marked as answer]
Probably not intended but probably fine, it's not really harming anyone.
